{"errors": [{"resource_id": "33891954", "parameter": "id", "resource_type": "user", "section": "data", "title": "Authorization Error", "value": "33891954", "detail": "Sorry, you are not authorized to see the user with id: [33891954].", "type": "https://api.twitter.com/2/problems/not-authorized-for-resource"}]}